src/frontend/src/lib/services/address.services.ts

```diff
@@ -302,6 +302,12 @@ const validateAddress = async <T extends Address>({
 	await warnSignOut(err ?? 'Error while certifying your address');
```

Comment:
Do you think we should pay a special attention to the fact that potentially both functions fail and call `warnSignOut`? I guess the first win given that sign-out reload the screen but, just wanted to know what you think.

Reply:
Glad to hear you share the same understanding. Let's keep it as it then, sounds good!